### DISCUSSION TITLE
Superslicer OSX failed

### DISCUSSION THREAD

**Discussion by MrMattn** [supermerill/SuperSlicer#3714] [Community contribution]
Hi
I have installed it on my macbook pro

but I can not open it!?
 
Can you help please?

screenshot:
https://ibb.co/Yyk9Gzw

**Comment by stanpete78** [comment]
Sorry for asking but did you do "xattr -cr /Applications/SuperSlicer.app" in Terminal?

**Reply by MrMattn** [reply]
hm no...should I have done that? :-)

**Reply by stanpete78** [reply]
It's not the typical error that I got, but you gotta do it anyways so it might as well be the solution.
If your Superslicer App is in "Programme" then you open your Terminal App and just write:

xattr -cr /Applications/SuperSlicer.app

then press enter. This should give the app the permissions it needs to start.

**Reply by MrMattn** [reply]
thanks, but didn't do anything
